content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -35,13 +35,13 @@ To generate service-level metrics directly in the Envoy proxies, follow these st
    A custom filter handles this exchange. Enable the metadata exchange filter with the following command:
 
     {{< text bash >}}
-    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/release-1.3/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
     {{< /text >}}
 
 1. To actually generate the service-level metrics, you must apply the custom stats filter.
 
     {{< text bash >}}
-    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/stats_filter.yaml
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/release-1.3/extensions/stats/testdata/istio/stats_filter.yaml
```

Comment:
Does this need to be pinned to 1.3? I suspect what you want instead is:

```suggestion
    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/{{< source_branch_name >}}/extensions/stats/testdata/istio/stats_filter.yaml
```

This will point to the right place automatically as releases progress.

Reply:
Ah, thanks! Great to know this. I also created a pr to master: https://github.com/istio/istio.io/pull/5189